tgui/packages/tgui/interfaces/NtosNetDownloader.tsx

```diff
@@ -20,31 +47,36 @@ export const NtosNetDownloader = (props, context) => {
     categories,
     programs,
```

Comment:
```
categories = [],
programs = [],
```
Please. This will stop the ui from outright crashing in the event there's a problem with the data